libs/langgraph/langgraph/pregel/messages.py

```diff
@@ -153,32 +153,35 @@ def on_chain_end(
         **kwargs: Any,
     ) -> Any:
         if meta := self.metadata.pop(run_id, None):
-            if isinstance(response, BaseMessage):
```

Comment:
Do we really need to make this recursive? Commands, when used, are always top level return values no?

Reply:
they can be returned inside lists - tool node returns a list of commands, for example